lib/Conversion/SeqToSV/FirRegLowering.cpp

```diff
@@ -70,6 +100,17 @@ void FirRegLowering::addToIfBlock(OpBuilder &builder, Value cond,
   }
 }
 
+FirRegLowering::FirRegLowering(TypeConverter &typeConverter,
+                               hw::HWModuleOp module,
+                               bool disableRegRandomization,
+                               bool emitSeparateAlwaysBlocks)
+    : typeConverter(typeConverter), module(module),
+      disableRegRandomization(disableRegRandomization),
+      emitSeparateAlwaysBlocks(emitSeparateAlwaysBlocks) {
+
+  reachableMuxes = std::make_unique<ReachableMuxes>(module);
```

Comment:
What's going on here re:unique_ptr, could this just be initialized like the others and a simple member?

(might be missing something?)

Since went to look, okay so these are created for each module and used in parallel and dropped at end of each module's processing.